osbs/build/build_request.py

```diff
@@ -317,6 +317,11 @@ def render_orchestrate_build(self):
 
         self.dj.dock_json_set_arg(phase, plugin, 'config_kwargs', config_kwargs)
 
+        goarch = {}
+        for platform, architecture in self.platform_descriptors.items():
+            goarch[platform] = architecture
+        self.dj.dock_json_set_arg(phase, plugin, 'goarch', goarch)
+
```

Comment:
Also need to set up OpenShift params: `url=None, verify_ssl=True, use_auth=True,`

Reply:
right will add, thanks